test/CheckoutSdkTest/Workflows/Four/WorkflowEventsTest.cs

```diff
@@ -40,18 +40,21 @@ List<EventTypesResponse> eventTypesResposes
             }
         }
 
-        [Fact(Timeout = 180000, Skip = "There is no payment by payment id and capture request")]
+        [Fact]
         public async Task ShouldGetSubjectEventAndEvents()
         {
+            CleanupWorkflowsForTesting();
```

Comment:
What do you think about doing the clean-up after test execution? That way we could avoid the `Nap`.